aws/alb-accesslog-table/main.tf

```
@@ -219,6 +222,11 @@ resource "aws_glue_catalog_table" "t" {
       name = "traceability_id"
```

fix field name from `traceability_id` to `conn_trace_id` .

```suggestion
      name = "conn_trace_id"
```

> The connection traceability ID is a unique opaque ID used to identify each connection. After a connection is established with a client, subsequent requests from this client will contain this ID in their respective access log entries. This ID acts as a foreign key to create a link between the connection and access logs.